packages/flutter/lib/src/widgets/localizations.dart

```diff
@@ -97,6 +99,12 @@ abstract class LocalizationsDelegate<T> {
   /// const constructors so that they can be used in const expressions.
   const LocalizationsDelegate();
 
+  /// Return true if the instance of 'T' loaded by this delegate's [load]
+  /// method supports the given `locale`'s language.
+  bool isSupported(Locale locale) => true;
+  // TODO(hansmuller): remove the default implementation after existing
+  // apps have had a chance to update.
```

Comment:
unless you mean that you plan to remove it in a couple of days, i would remove it now. the longer we leave this the more likely it is we'll have people depending on it.